tests/unit_test/channels/mail_scheduler_test.py

```
@@ -108,3 +108,69 @@ def test_schedule_channel_mail_reading_exception(mock_mongo_client, mock_mail_pr
+    mock_execute_http_request.return_value = {'success': True}
+    mock_imp.return_value = True
+    bot = "test_bot"
+    try:
+        MailScheduler.request_stop(bot)
+    except AppException:
+        pytest.fail("request_epoch() raised AppException unexpectedly!")
+
+@patch('kairon.shared.channels.mail.processor.MailProcessor.does_mail_channel_exist')
+@patch('kairon.shared.channels.mail.scheduler.Utility.get_event_server_url')
+@patch('kairon.shared.channels.mail.scheduler.Utility.execute_http_request')
+def test_request_stop_success(mock_execute_http_request, mock_get_event_server_url, mock_imp):
```

Ruff (0.8.2)

Redefinition of unused `test_request_stop_success` from line 132

(F811)